cmd/init.go

```diff
@@ -45,24 +44,32 @@ func runInit(args ...string) {
 
 	// Create repo
 	checkErr(os.MkdirAll(homedir, 0700))
-	initRenter(homedir)
+	initRenter(homedir, *keyFileFlag)
 	initProvider(homedir)
 }
 
-func initRenter(homedir string) {
+func initRenter(homedir string, keyfile string) {
```

Comment:
Is `homedir` referring to the user's home directory, or SkyBin's? If it's referring to SkyBin's, I think we might want to rename that so it's less ambiguous.

Reply:
It's the skybin homedir. I agree it's ambiguous.